src/test/java/klerer/rijks/RijksServiceTest.java

```diff
@@ -0,0 +1,61 @@
+package klerer.rijks;
+
+import klerer.rijks.json.RijksCollection;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RijksServiceTest {
+    @Test
+    public void pageNum() {
```

Comment:
If these tests run locally, but don't run on GitHub then you haven't set up your API key on GitHub correctly.

Reply:
my tests fail locally